.circleci/config.yml

```diff
@@ -32,7 +32,7 @@ jobs:
 
       - run:
           name: "compile"
-          command: "mvn clean install"
+          command: "mvn clean install -Dnet.bytebuddy.experimental=true"
```

Comment:
what is the experimental thing for?

Reply:
One of the dependencies (bytebuddy from Mockito) won't work with Java 21 without it.  But let me try to update Mockito and see if that fixes it.

Reply:
Phew!  😅 It wasn't easy but I finally got the right set of dependencies so we don't need that flag.  I think the "-Dnet.bytebuddy.experimental=true" was temporarily needed for Java 21 until recently.